draft-art-tigress.md

```diff
@@ -71,225 +71,205 @@ normative:
 
 --- middle
 
 # Introduction
 
-Today, there is no standard way of transferring digital credentials securely between two devices
-belonging to the same platform or two different platforms. This document proposes a solution to this problem
-by introducing a Relay server which allows two devices to exchange encrypted Provisioning Information securely.
-The Relay server solves this problem by creating and managing temporary mailbox storage.
+Mobile devices with ever increasing computational power and security capabilities are enabling various use cases. One such category includes use of mobile devices to gain access to a property that a user owns or rents or is granted access to. The cryptographic material and other data required to enable this use case is termed as Digital Credential. The process of getting a Digital Credential on a mobile device is termed as Provisioning.
 
-Each mailbox can be referenced by devices using a unique mailbox identifier in a URL.
-The URL pointing to encrypted Provisioning Information is to be passed between devices directly
-over various channels (e.g. SMS, email, messaging applications).
-The Security Considerations section provides recommendations on passing the URL and the Secret securely.
+Based on type of property, different public or proprieatary standards (termed as Verticals) govern details of Digital Credentials used to access them. These details include policies, mechanism and practices to create, maintain and use Digital Credentials and vary considerably across Verticals.
 
-This document describes a Hypertext (HTTP) Application Programming Interface (API) that allows
-Sender and Receiver devices to interact with a Relay server in order to perform secure credential transfer.
+Once a user has a Digital Credential for some Vertical provisioned on their mobile device, next natural use case is to share it with others. Sharing a Credential should feel like a natural extension of regular communication methods (like instant messaging, sms, email). The user experience of sharing a Credential should be intuitive, similar to sharing other digital assets like photos or documents. The sharing process should be secure and privacy preserving.
 
+Credentials pose two requirements that differ from sharing other digital assets. The Initiator and Recipient devices may need to communicate back and forth to get the necessary Provisioning Information. The Provisioning information exchange must be limited to Initiator device and the first Recipient device to claim the information.
 
-# Terminology
+To achive these goals, a new transport is necessary. This documents describes a Hypertext (HTTP) Application Programming Interface (API) to create such a transport termed as Relay Server. The document also defines JSON based data to enable a uniform user experience for securely sharing Digital Credentials of various types.
```

Comment:
... This document specifies an API for a transport protocol built using standard HTTP [RFC9110].
The document also defines data in JSON standard [RFC8259] to enable a uniform user experience for securely sharing Digital Credentials of various types.